### PR TITLE
Google search for image sends screenshot of webpage successfully added.

### PR DESCRIPTION
**Pull request by SinghHimadri**
## Issue that this pull request solves
#330 
Closes: # (issue number)

## Proposed changes

Brief description of what is fixed or changed
The target can write /image {query} and the bot will automatically search google images for the query and save that image in wplay/chatbotImages folder.Now the user will automatically be redirected to choose file user can choose the then the image screenshot is sent to the target. The function has been made compatible with pyppeteer the browser is open seacges google image takes screenshot and saves it.
## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Screenshot from 2020-05-18 00-40-18
The screenshot saved in wplay/chatbotfolder
Screenshot from 2020-05-18 00-48-38
User redirected automatically to choose file
Screenshot from 2020-05-18 00-39-55

WhatsApp Image 2020-05-18 at 12 51 26 AM

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- wplay/chatbot.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5538445&bid=18155588)

File: wplay/chatbot.py [hunk 1]
```diff
@@ -1,5 +1,9 @@
 from googlesearch import search
-def Bot(last_Message):
+from wplay.utils import io
```
Comment (rpotter12) [inline wplay/chatbot.py:2]: Codacy Issue found: [Unused io imported from wplay.utils](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5538445)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/chatbot.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5538445&bid=18155588)

**Comment by SinghHimadri** [comment]
@rpotter12 Please review my PR.

**Pull request merged**